apps/aechannel/src/aesc_fsm.erl

```diff
@@ -3202,7 +3185,7 @@ log(Op, Type, M, #data{log = Log0} = D) ->
     D#data{log = log_msg(Op, Type, M, Log0)}.
 
 log_msg(Op, Type, M, Log) ->
-    aesc_window:add({Op, Type, os:timestamp(), M}, Log).
+    aesc_window:add_new({Op, Type, os:timestamp(), M}, Log).
```

Comment:
Given that the added item contains an `os:timestamp()` entry (microsecond timestamp), what are the odds that duplicates will be detected and handled?

Reply:
Odds are low in this case, but I figured its the right thing to do semantically.